src/App.js

```diff
@@ -45,10 +45,18 @@ class Navbar extends React.Component {
 class SelectionBox extends React.Component {
```

Comment:
What exactly is the selection box here? Do you mean box containing all the testimonies?

Reply:
Selection box is the box that has dropdown options for the user to find a tutor
![image](https://user-images.githubusercontent.com/29843006/41006058-d0d6da2e-68d4-11e8-8166-bbfd18d77422.png)